Dockerfile

```diff
@@ -39,31 +36,158 @@ RUN conda install --name python2 --quiet --yes \
+RUN echo "deb-src http://deb.debian.org/debian testing main" >> /etc/apt/sources.list
+RUN echo "deb http://ftp.debian.org/debian jessie-backports main" >> /etc/apt/sources.list
+RUN apt-get update && \
+    apt-get -y install libssl-dev libxml2-dev libcurl4-openssl-dev
+
+#########################################################################################
+## Necessary for running maven
+## copied from https://hub.docker.com/r/cardcorp/r-java/~/dockerfile/
+##
+## gnupg is needed to add new key
+RUN apt-get update && apt-get install -y gnupg2
+
+## Install Java 
+RUN apt-get update \
+    && apt-get install -t jessie-backports -y openjdk-8-jdk \
+    && echo "MAVEN IS NOT IN THE UPSTREAM LIST (JOSH)" \
+    && apt-get install -y maven \ 
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean 
+
+ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+RUN rm -rf /usr/lib/jvm/java
+RUN ln -s  /usr/lib/jvm/java-8-openjdk-amd64 /usr/lib/jvm/java
+
+RUN /opt/conda/envs/python2/bin/conda install --quiet --yes -c r r
+RUN R CMD javareconf
+    
+## make sure Java can be found in rApache and other daemons not looking in R ldpaths
+RUN echo "/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/server/" > /etc/ld.so.conf.d/rJava.conf
+RUN /sbin/ldconfig
+
+###
+### Fix install2.r
+### --------------
+### See https://github.com/rocker-org/rocker/blob/e9758030e435915d5e6f21aaab0fc35a5a8efaae/r-base/Dockerfile#L41
+### https://github.com/rocker-org/rocker/issues/149
+### --------------
+## Now install R and littler, and create a link for littler in /usr/local/bin
+## Also set a default CRAN repo, and make sure littler knows about it too
+RUN apt-get update \
+        && echo "MIGRATE THIS BLOCK TO THE APT-GET ABOVE" \
+    && apt-get install -y --no-install-recommends \
+        littler \
+        && echo 'options(repos = c(CRAN = "https://cran.rstudio.com/"), download.file.method = "libcurl")' >> /etc/R/Rprofile.site \
+        && echo 'source("/etc/R/Rprofile.site")' >> /etc/littler.r \
+    && ln -s /usr/share/doc/littler/examples/install.r /usr/local/bin/install.r \
+    && ln -s /usr/share/doc/littler/examples/install2.r /usr/local/bin/install2.r \
+    && ln -s /usr/share/doc/littler/examples/installGithub.r /usr/local/bin/installGithub.r \
+    && ln -s /usr/share/doc/littler/examples/testInstalled.r /usr/local/bin/testInstalled.r \
+    && install.r docopt \
+    && rm -rf /tmp/downloaded_packages/ /tmp/*.rds \
+    && rm -rf /var/lib/apt/lists/*
+
+## Install rJava package
+ENV PATH=$PATH:/usr/lib/jvm/java-8-openjdk-amd64/bin
+RUN conda install --quiet --yes -c r r-rjava=0.9_8
+
+# Changed from rOMERO-gateway/Dockerfile
+RUN chown jovyan /usr/local/lib/R/site-library
+
+##
+##
+#########################################################################################
+
+USER root
+
+# install romero
+ENV _JAVA_OPTIONS="-Xss2560k -Xmx2g"
+RUN /opt/conda/envs/python2/bin/conda install --quiet --yes -c anaconda gfortran_linux-64
```

Comment:
What requires `gfortran_linux-64`? Is `anaconda` needed?

Reply:
Yes this compiler was required for compiling the dependencies for r-java (once the base image was changed)